src/engine/strat_engine/thinpool/filesystem.rs

```diff
@@ -230,5 +230,6 @@ impl Recordable<FilesystemSave> for StratFilesystem {
 pub fn fs_usage(mount_point: &Path) -> EngineResult<(Bytes, Bytes)> {
     let mut stat = Statvfs::default();
     statvfs(mount_point, &mut stat)?;
-    Ok((Bytes(stat.f_bsize * stat.f_blocks), Bytes(stat.f_bsize * (stat.f_blocks - stat.f_bfree))))
+    Ok((Bytes(stat.f_bsize as u64 * stat.f_blocks),
```

Comment:
I think it's worth casting the value in one place and putting a comment explaining why.